Makefile

```diff
@@ -0,0 +1,23 @@
+all: backend frontend
+
+backend:
+	docker build . -f docker/Dockerfile_server
```

Comment:
Nie trzeba osobno budować Dockera, chyba że z jakiegoś powodu chcecie / potrzebujecie. Docker Compose potrafi to zbudować sam, trzeba mu tylko podać konfigurację: https://docs.docker.com/compose/compose-file/build/#build-definition i podać `--build`